ko/sql/query/delete.rst

```diff
@@ -12,16 +12,18 @@ DELETE
 ::
  
     <DELETE single table>
-    DELETE [FROM] table_name [ WHERE <search_condition> ] [LIMIT row_count]
+    DELETE [FROM] [schema_name.]table_name [ WHERE <search_condition> ] [LIMIT row_count]
      
     <DELETE multiple tables FROM ...>
-    DELETE table_name[, table_name] ... FROM <table_specifications> [ WHERE <search_condition> ]
+    DELETE [schema_name.]table_name [{, [schema_name.]table_name}] FROM <table_specifications> [ WHERE <search_condition> ]
```

Comment:
```suggestion
    DELETE [schema_name.]table_name ...  FROM <table_specifications> [ WHERE <search_condition> ]
```

Reply:
The reader may be misunderstood that only 1 or 2 table names can be specified.

Reply:
Thanks, I'll change it.